src/main/java/com/epam/prd/jjop/helpers/arrays/Copy.java

```diff
@@ -0,0 +1,184 @@
+         * @param dest_arr - array to be copied in.
+         * @param destPos - starting position in the destination data, where to copy in.
+         * @param len -the number of array elements to be copied.
+         */
+
+        /**
+         * Uses method java.lang.System.arraycopy() to copy part of Source array to Destination one.
+         */
+        //Prints source and destination arrays
+        System.out.println("arraySource: " + Arrays.toString(arraySource));
+        System.out.println("arrayDestination: "
+                + Arrays.toString(arrayDestination));
+        System.arraycopy(arraySource, 1, arrayDestination, 2, 3);
+        System.out.println("arrayDestination after arrayCopy: "
+                + Arrays.toString(arrayDestination));
+        /**
+         * Uses method java.lang.System.arraycopy() to copy Source array to Source array with overlapping areas
+         */
+        System.out.println("Before coping: " + Arrays.toString(intsArray));
+        System.arraycopy(intsArray, 1, intsArray, 3, 3);
+        System.out.println("After arraycopy: " + Arrays.toString(intsArray));
+
+        /**
+         * Secondly lets look at possibilities of java.util.Arrays.copyOf methods that
+         * Copy the specified array, truncating or padding with the default values (if necessary) so the copy has the specified length.
+         * <p>There are 8 overloaded types of java.util.Arrays.copyOf of primitive array
+         * e.g. boolean, int, short, char, byte, long, float, and double as well as reference type array (see below).</p>
+         * <b>Syntax</b>:
+         * <p>copyOf(int[] original, int newLength)</p>
+         * @param original – the array to be copied.
+         * @param newLength – the length of the copy to be returned.
+         * @throws NegativeArraySizeException if newLength is negative.
+         * @throws NullPointerException if original is null.
+         * @throws ArrayStoreException - if an element copied from original is not of a runtime type that can be stored in an array of class newType.
+         *
+         * Arrays.copyOf methods for reference type array:
+         * <ul>
+         *    <li> {@link java.util.Arrays#copyOf(Object[], int)} - Copies the specified array, truncating or padding with null
+         *          characters (if necessary) so the copy has the specified length.
+         *    </li>
+         *    <li> {@link java.util.Arrays#copyOf(Object[], int, Class)} - Copies the specified array, truncating or padding with null
+         *         characters (if necessary) so the copy has the specified length.
+         *    </li>
+         *  </ul>
+         */
+        /**
+         * Prints copies of different types arrays
+         * New array elements will be assigned from an existing value from an old array.
+         * If the length of the new array is greater than the old array in such cases all remaining elements
+         * will be assigned as zero or null.
+         */
```

Comment:
Same as above. Javadoc should be directly above the things described by it (without the empty line in between).